### ISSUE TITLE
[BUG/ISSUE] RRTMG set to false in HEMCO_Config.rc file that ships with the unit tester

### ISSUE THREAD

**Issue by bsuther** [geoschem/geos-chem#235]
## Describe the bug
I am getting an unclear error message "RRTMG is set to false in HEMCO_Config.rc but should be set to false for this simulation." which seems to contradict itself, so I am not sure how to respond to it. It is also saying to look in the hemco log file for more information, but it stops before creating HEMCO.log.

## To Reproduce
Compile version 12.7.1 with rrtmg turned on and attempt to run it. 

**Compilation commands**
make -j4 build RRTMG=y

**Run commands**
./geos

## Expected behavior
The model to run with rrtmg turned on. 

## Error messages
```
===============================================================================

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%               HEMCO: Harvard-NASA Emissions Component               %%%%%
%%%%%               You are using HEMCO version 2.2.0                     %%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
  
Reading settings & switches of HEMCO configuration file: HEMCO_Config.rc
===============================================================================
GEOS-Chem ERROR: RRTMG is set to false in HEMCO_Config.rc but should be set to 
false for this simulation.
 -> at CheckSettings (in module GeosCore/hcoi_gc_main_mod.F90)
===============================================================================

===============================================================================
GEOS-Chem ERROR: Error encountered in "CheckSettings"!
 -> at HCOI_GC_Init (in module GeosCore/hcoi_gc_main_mod.F90)

THIS ERROR ORIGINATED IN HEMCO!  Please check the HEMCO log file for 
additional error messages!
===============================================================================

===============================================================================
GEOS-Chem ERROR: Error encountered in "HCOI_GC_Init"!
 -> at Emissions_Init (in module GeosCore/emissions_mod.F90)
===============================================================================

===============================================================================
GEOS-CHEM ERROR: Error encountered in "Emissions_Init"!
STOP at  -> at GEOS-Chem (in GeosCore/main.F)
===============================================================================

```

## Required information
Please include the following:
 - GEOS-Chem version you are using: 12.7.1
 - Compiler version that you are using: gfortran 9.2.0
 - netCDF and netCDF-Fortran library version: 4.6.3-gcc4.8.5
 - Computational environment: cluster
 - Are you using "out of the box" code, or have you made modifications?
 Mainly out of the box, but I modified line 64 of GeosRad/Makefile from
OPT     := -O1 -override-limits
to
OPT += -fno-var-tracking-assignments
because I was previously getting the error 
"-DBPCH_TPBC -DRRTMG -DNC_HAS_COMPRESSION -DUSE_REAL8 -J../mod
-I/usr/local/apps/netcdf-centos7/4.6.3-gcc4.8.5/include -fdefault-real-8
-fdefault-double-8 -c -O1 -override-limits rrtmg_sw_rad.F90
rrtmg_lw_k_g.F90:4298:0:

 4298 |       subroutine lw_kgb03
      |
note: variable tracking size limit exceeded with
‘-fvar-tracking-assignments’, retrying without
ar crs librad.a mcica_subcol_gen_lw.o ..."

## Input and log files to attach
[input_and_log_files.zip](https://github.com/geoschem/geos-chem/files/4289922/input_and_log_files.zip)

## Additional context
Include any other context about the problem here.

**Comment by yantosca** [comment]
This might be a compiler issue.  GC 12.7.0 has some problems with the GNU Fortran compilers prior to gfortran 6.  You might want to try again with a more recent GNU Fortran version, if possible.

**Comment by yantosca** [comment]
Oh I see you are using gfortran 9.2.0.  Nevermind.

**Comment by yantosca** [comment]
I think I see the issue.  The HEMCO_Config.rc file has RRTMG set to false, but should be true.  
```
# ----- NON-EMISSIONS DATA ------------------
    --> UVALBEDO               :       true
    --> CCM_STRAT_Bry          :       true
    --> GMI_STRAT_OH           :       true
    --> GMI_PROD_LOSS          :       false
    --> UCX_PROD_LOSS          :       true
    --> OMOC_RATIO             :       false
    --> GMD_SFC_CH4            :       true
    --> CMIP6_SFC_CH4          :       false
    --> OLSON_LANDMAP          :       true
    --> YUAN_MODIS_LAI         :       true
    --> RRTMG                  :       false
    --> SfcVMR                 :       false
```
I will doublecheck if this is an error in the HEMCO_Config.rc that is created in the RRTMG run directory.

**Comment by bsuther** [comment]
RRTMG was not one of the non-emissions data fields in the previous version I was running (12.6.1). Can you clarify for me what turning it on will do? Does turning it on mean it will be using a data set for rrtmg values rather than calculating them online? Thank you for your help.

**Comment by yantosca** [comment]
HEMCO reads in initialization data files for RRTMG.

Also, I've confirmed that the HEMCO_Config.rc in the RRTMG run directory has RRTMG set to false.  That is a bug, it should be set to true.  This may have come in when we did a reorganization and cleanup of the HEMCO_Config files in 12.7.0.  I would turn RRTMG to true in HEMCO_Config.rc and that should let your run proceed.   We will make a fix for this in 12.7.2.

Also note: right now, we don't have an official liaison for the RRTMG simulation in the GEOS-Chem community.  We are working on identifying someone who can answer RRTMG-related science questions.  Stay tuned.
